.github/workflows/gemini-invoke.yml

```
@@ -0,0 +1,122 @@
+          gcp_project_id: '${{ vars.GOOGLE_CLOUD_PROJECT }}'
+          gcp_service_account: '${{ vars.SERVICE_ACCOUNT_EMAIL }}'
+          gcp_workload_identity_provider: '${{ vars.GCP_WIF_PROVIDER }}'
+          gemini_api_key: '${{ secrets.GEMINI_API_KEY }}'
+          gemini_cli_version: '${{ vars.GEMINI_CLI_VERSION }}'
+          gemini_debug: '${{ fromJSON(vars.DEBUG || vars.ACTIONS_STEP_DEBUG || false) }}'
+          gemini_model: '${{ vars.GEMINI_MODEL }}'
+          google_api_key: '${{ secrets.GOOGLE_API_KEY }}'
+          use_gemini_code_assist: '${{ vars.GOOGLE_GENAI_USE_GCA }}'
+          use_vertex_ai: '${{ vars.GOOGLE_GENAI_USE_VERTEXAI }}'
+          upload_artifacts: '${{ vars.UPLOAD_ARTIFACTS }}'
+          workflow_name: 'gemini-invoke'
```

actionlint (1.7.9)

input "workflow_name" is not defined in action "google-github-actions/run-gemini-cli@v0". available inputs are "extensions", "gcp_location", "gcp_project_id", "gcp_service_account", "gcp_workload_identity_provider", "gemini_api_key", "gemini_cli_version", "gemini_debug", "gemini_model", "google_api_key", "prompt", "settings", "upload_artifacts", "use_gemini_code_assist", "use_vertex_ai"

(action)